media/css/sandstone/buttons.less

```diff
@@ -29,6 +31,16 @@
     }
 }
 
+.button-flat-green {
```

Comment:
@jpetto - I would give you +100 internet points to put this guy directly into master? :money_with_wings: